docs/changelog/116712.yaml

```diff
@@ -0,0 +1,10 @@
+pr: 116712
+summary: Remove apm_user role
+area: Security
+type: breaking
+issues: []
+breaking:
+  title: Remove deprecated `apm_user` role
+  area: Authorization
+  details: The deprecated `apm_user` role has been removed.
+  impact: Users depending on built-in `apm_user` role will lose access to APM UI and data. In order to retain access, replace it by one of the existing built-in roles ({ref}/built-in-roles.html#built-in-roles-viewer[viewer], {ref}/built-in-roles.html#built-in-roles-editor[editor] or {ref}/built-in-roles.html#built-in-roles-superuser[superuser]).
```

Comment:
```suggestion
  impact: Users depending on built-in `apm_user` role will lose access to APM UI and data. In order to retain access, replace it by one of the existing [built-in roles](elasticsearch/roles.md) `viewer`, `editor` or `superuser`).
```

Reply:
I think this link will resolve correctly once this gets injected into the new markdown file, but if we want to be super safe, can just remove the link entirely. 

Using markdown syntax because as of 9.0 we've [migrated to new system](https://elastic.github.io/docs-builder/syntax/)

Reply:
This is just to change asciidoc syntax, otherwise @slobodanadamovic's [rewording](https://github.com/elastic/elasticsearch/pull/125810#discussion_r2024644483) has my approval :)